p4-16/spec/P4-16-spec.mdk

```diff
@@ -3930,6 +3949,12 @@ methods are not defined for header unions.
 Supplying an expression with a union type to `emit` simply emits the
 single header that is valid, if any.
 
+The expression `u.minSizeInBits()` for a value `u` with a header union
+type is a compile-time constant value of type `int`.  It is equal to
+the maximum value of `h.minSizeInBits()` among all members `h` of `u`.
```

Comment:
Some might be surprised by this choice of taking the maximum of all members `h` of `u`. It may be worth adding a brief explanation of why this is the right choice.

Reply:
To me, this is obvious.   The code has been this way in p4c for a long time.

https://github.com/p4lang/p4c/blob/master/ir/type.def#L260